src/flashlsAudioTracks.js

```diff
@@ -0,0 +1,36 @@
+import videojs from 'video.js';
+
+/**
+ * Updates the selected index of the audio track list with the new active track
+ *
+ * @param {Object} tech
+ *        The flash tech
+ * @function updateAudioTrack
+ */
+export const updateAudioTrack = (tech) => {
+  const audioTracks = tech.audioTracks();
+
+  for (let i = 0; i < audioTracks.length; i++) {
+    if (audioTracks[i].enabled) {
+      tech.el_.vjs_setProperty('audioTrack', audioTracks[i].id);
```

Comment:
I think we should return after setting this property so it only sets the first enabled track if more than one is enabled. I believe the player can temporarily get in the state of more than one audio track being enabled when say you have track1 disabled and track2 enabled. When the user selects to switch to track1, videojs will first change track1 to enabled and trigger the `change` event. At this time both track1 and track2 will be enabled when this function runs. Now immediately after videojs will disable track2 and this function will get run again properly setting track1 as the active track. But its probably best to just set the property once during this function